build/update-tinygo.sh

```diff
@@ -9,11 +9,13 @@
 set -e
 set -x
 
-actor_sdk_version=3893510790129936fa561b386eedcec6c71338a0
-tinygo_msgpack_version=192ec93ec5c440e685d61c705c8fe68ff4aa1973
+actor_sdk_version=v0.1.1
+tinygo_msgpack_version=v0.1.4
+tinygo_cbor_version=v0.1.0
 
 WASH=${WASH:-wash}
-$WASH gen -c ./codegen-go.toml
+#$WASH gen -c ./codegen-go.toml
+../weld/target/debug/codegen ./codegen-go.toml
```

Comment:
Is this intended to still be just the debug weld or should it be back to `$WASH`?

Reply:
It won't work with wash yet until wash updates it's codegen dependency to 0.4.5 and has a new release

Reply:
Okay, I'm fine with this since it's a script we're likely only using internally